internal/interfaceadapter/repository/paypayrepo/repository.go

```diff
@@ -0,0 +1,75 @@
+package paypayrepo
+
+import (
+	"context"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/RettyInc/retty3-proto/go/services/externals"
+	"github.com/pkg/errors"
+)
+
+type paypayRepository struct {
+	connManager repository.ConnManager
+}
+
+func New(connManager repository.ConnManager) *paypayRepository {
+	return &paypayRepository{
+		connManager: connManager,
+	}
+}
+
+func (r *paypayRepository) GetPaypayShops(
+	ctx context.Context,
+	in *externals.PaypayShopsQuery,
+) (*externals.PaypayShopsResponse, error) {
+	log.Printf("Called GetShops: %v", in)
+
+	query := `
+		SELECT rp.restaurant_id, rp.paypay_merchant_id, rp.paypay_shop_id
+		FROM retty_paypay AS rp
+		WHERE rp.restaurant_id in (?)
+		AND rp.status = 1
+	`
+
+	restaurantIds := make([]string, len(in.RettyRestaurantIds))
+	for i, v := range in.RettyRestaurantIds {
+		restaurantIds[i] = strconv.FormatUint(v, 10)
+	}
+
+	rows, err := r.connManager.GetSlave().QueryContext(
+		ctx,
+		query,
+		strings.Join(restaurantIds, ","),
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "DB Access Error")
```

Comment:
pkg/errors をやめて fmt.Errorf 使うように揃えましょう
```suggestion
		return nil, fmt.Errorf("db access error: %w", err) 
```

Reply:
承知しました！直します